jenkins/new-scan-engine/README.md

```diff
@@ -30,4 +30,4 @@ The [example pipeline](Jenkinsfile-jenkins-plugin) shows how to use it to build
 Both approaches require a couple of things:
 
 * A valid Sysdig Secure API token
```

Comment:
+ "The appropriate Sysdig vulnerability scanning endpoint depending on your region"
https://docs.sysdig.com/en/docs/administration/saas-regions-and-ip-ranges

Reply:
Thanks! I've made some changes, PTAL